api/DOMTokenList.json

```
@@ -38,7 +38,7 @@
             "version_added": "1.0"
           },
           "webview_android": {
-            "version_added": "1"
+            "version_added": "≤37"
```

```suggestion
            "version_added": "3"
```

Based on the fact that WebView 3 uses WebKit 534.13.

---

May I ask where you sourced the information about WebView's engine versions?

---

I figured it out from https://wiki.developer.mozilla.org/en-US/docs/MDN/Contribute/Processes/Matching_features_to_browser_version#Safari and updated that a bit in the process. Basically, I have a checkout of WebKit where I can find the right commit easily using `git log -S` and other git tricks. Given the commit, get the approximate WebKit version from Source/WebCore/Configurations/Version.xcconfig, and from there determine if the right releases can be determined or if the exact WebKit version has to be found. That last bit I haven't done, it would involve stepping through branches and looking at the content of each. Would be much faster with the branches in a local checkout, but I don't have that.

---

Oh, and of course, with this approach one has to check if the feature was behind an ifdef or a runtime flag. If it is then I wouldn't have confidence in determining the release for Safari. (I do know how Chromium controls features well enough to do this though.)